assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -78,21 +78,9 @@ export default function PropertySelect( props ) {
 			return false;
 		}
 
-		const isResolvingProperties =
```

Comment:
It turns out we need to wait for both `PropertySelect` and `WebDataStreamSelect` to finish loading as part of the loading state logic in `SettingsEnhancedMeasurementSwitch`. To support that, I've extracted this loading logic into a reusable `isLoadingProperties()` selector.